includes/Traits/Types.php

```diff
@@ -154,6 +155,24 @@ public function get_allowed_image_mime_types() : array {
 		return array_values( array_intersect( $allowed_mime_type, $image_mime_type ) );
 	}
 
+	/**
+	 * Returns a list of audio mime types.
+	 *
+	 * Used for the allowlist when adding background audio.
+	 *
+	 * @since 1.10.0
+	 *
+	 * @return array List of allowed mime types.
+	 */
+	public function get_allowed_audio_mime_types(): array {
+		return [
+			'audio/mpeg',
+			'audio/aac',
+			'audio/wav',
+			'audio/ogg',
```

Comment:
Are all these mime types web safe?

Reply:
`audio/mpeg` is MP3 and works in all browsers.

`audio/aac` works everywhere, but in Firefox it depends on the OS.

`audio/wav` works everywhere except for IE 11, which we don't support.

`audio/ogg` works everywhere, but in Safari only on macOS 11.3 or higher.

So yeah I'd consider them all safe.